sites/svelte-5-preview/src/routes/docs/content/03-appendix/02-breaking-changes.md

```diff
@@ -133,6 +133,23 @@ In Svelte 4 syntax, every property (declared via `export let`) is bindable, mean
 
 Setting the `accessors` option to `true` makes properties of a component directly accessible on the component instance. In runes mode, properties are never accessible on the component instance. You can use component exports instead if you need to expose them.
 
+### Props may update with same value
+
+Svelte 4 did safe equality checks on properties by default. That meant that even if you update a value, and part of of tha value is a primitive passed as a property, and that part did not change, it would not trigger a property update.
```

Comment:
This section reads a little confusingly to me, and it contains typos ('of of tha') — let's move it into a separate PR